TLM/TLM/Manager/Impl/LaneConnection/LaneConnectionSubManager.cs

```diff
@@ -423,6 +452,15 @@ public class LaneConnectionSubManager :
             return true;
         }
 
+        private void AssertLane(uint laneId, bool startNode) {
```

Comment:
As this function ends with `Assert` call, is that a debug assert? And in release will it do the calculations but no assert?
Declare this function as `[Conditional("DEBUG")]`?

Reply:
its only called when adding/removing lane connections so its not performance critical. therefore I decided to include it in release build. you never know .... it might help with debugging user errors.

Should I make it conditional?

Reply:
I mean the assert function is marked debug only. i believe it is. So in release this code does nothing, just calculates values and returns.

Reply:
Assert is not marked as debug. 
`AssertNotNone` is the only one that is marked as debug. its inconsistent with all the rest of the assert functions.
but that does not matter. I think its a good idea to execute assert in debug mode here.

Reply:
I removed the `[Conditional("DEBUG")]` from `AssertNotNone()` and instead added #if DEBUG around the usage. so now its consistent